compiler/rustc_codegen_llvm/src/abi.rs

```diff
@@ -444,15 +413,14 @@ impl<'ll, 'tcx> FnAbiLlvmExt<'ll, 'tcx> for FnAbi<'tcx, Ty<'tcx>> {
     }
 
     fn apply_attrs_llfn(&self, cx: &CodegenCx<'ll, 'tcx>, llfn: &'ll Value) {
-        // FIXME(eddyb) can this also be applied to callsites?
+        let mut func_attrs = SmallVec::<[_; 2]>::new();
         if self.ret.layout.abi.is_uninhabited() {
-            llvm::Attribute::NoReturn.apply_llfn(llvm::AttributePlace::Function, llfn);
+            func_attrs.push(llvm::AttributeKind::NoReturn.create_attr(cx.llcx));
         }
-
-        // FIXME(eddyb, wesleywiser): apply this to callsites as well?
         if !self.can_unwind {
-            llvm::Attribute::NoUnwind.apply_llfn(llvm::AttributePlace::Function, llfn);
+            func_attrs.push(llvm::AttributeKind::NoUnwind.create_attr(cx.llcx));
         }
+        attributes::apply_to_llfn(llfn, llvm::AttributePlace::Function, &{ func_attrs });
```

Comment:
Why does this need braces?

Reply:
Not necessary, but causes the SmallVec to be moved, which makes `func_attrs.push()` after this point an error.